mailu/templates/dovecot.yaml

```diff
@@ -145,4 +154,17 @@ spec:
     port: 4190
     protocol: TCP
 
+---
+
+{{- if .Values.dovecot.overrides }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: override-dovecot
```

Comment:
Please use the template for the name like in:
https://github.com/Mailu/helm-charts/blob/90d2e47ea62e5be4207b4aaa131fa668d0a2ee10/mailu/templates/dovecot.yaml#L8

Reply:
Also, please update the chart version to 0.1.1. The rest looks very good, I'd merge it then.